README.md

```diff
@@ -171,6 +172,7 @@ navPrev: PropTypes.node,
 navNext: PropTypes.node,
 onPrevMonthClick: PropTypes.func,
 onNextMonthClick: PropTypes.func,
+onClose: ({ startDate, endDate }) => this.setState({ startDate, endDate }) // PropTypes.func,
```

Comment:
English text in the documentation. propTypes are not for a human, they're for React to issue warnings when it doesn't match.